tests/CMakeLists.txt

```diff
@@ -9,11 +9,19 @@ add_subdirectory(ondemand)
 #
 if(NOT SIMDJSON_LEGACY_VISUAL_STUDIO AND NOT SIMDJSON_WINDOWS_DLL)
   add_cpp_test(numberparsingcheck LABELS acceptance per_implementation)
-  target_link_libraries(numberparsingcheck simdjson-include-source simdjson-windows-headers)
+  target_link_libraries(numberparsingcheck simdjson simdjson-windows-headers)
```

Comment:
I'm curious how this works ... I would have thought using -Lsimdjson and simultaneously #including simdjson.cpp would cause duplicate link errors?

Reply:
Maybe that's where the ODR violations come from in 2 of the CircleCI workflows? Things are a messy and it's very hard to understand dependencies between the targets.

Reply:
Ah. I think it did.